l10n/en-US/viewer.ftl

```diff
@@ -326,6 +326,14 @@ pdfjs-editor-stamp-button-label = Add or edit images
 pdfjs-editor-remove-button =
     .title = Remove
```

Comment:
Shouldn't this entry be removed now, since it's no longer used?

Reply:
If it's unused (and it looks like it), indeed it should be removed